docs/services/ml/upload-training-script.md

```diff
@@ -390,7 +412,7 @@ You can use the [`viam training-script upload`](/cli/#training-script) command t
 For example:
 
 ```sh {class="command-line" data-prompt="$"}
-viam training-script upload --path=<path-to-tar.gz> --org-id=<INSERT ORG ID> --script-name="MyCustomTrainingScript"
+viam training-script upload --path=<path-to-tar.gz> --org-id=<INSERT ORG ID> --script-name=<org-id-or-namespace:training-script-name>
```

Comment:
I thought having string was helpful in example to show that's how you do it-- maybe tabs here could be helpful?